src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Can you add a comment to this file indicating its purpose (if this is possible).

Reply:
I will see what is possible. The comment will be something like "This Mockito extension allows us to mock final methods, which are very common in OpenJFX".

Reply:
Interesting.!

At least in earlier Mockito documention it is very clear that exactly one line is expected in this file, which suggesting commenting is not possible. However, it turns out OK to have the comment starting with a '#' character. Double slash causing mockito to fail to initialize as it tries to read the comment as an extension. Commenting out the extension itself with a '#' does what you'd expect -- disables the extension causing the test that requires mocking of a final method to fail.

I might approach the Mockito community and ask them to document this feature, or point me in the right direction if I missed the documentation somehow, as it's very useful to explain what the file does! It would give us peace of mind, if we knew it was fully intended.

Reply:
Comment added and pushed